src/interaction.js

```diff
@@ -16,21 +16,31 @@ limitations under the License.
 
 import {UserInteractionPlugin} from '@opentelemetry/plugin-user-interaction';
 
-const allowedEventTypes = {
-  click: true,
-  dblclick: true,
-  submit: true,
-  reset: true,
-  dragend: true,
-  drop: true,
-  ended: true,
-  pause: true,
-  play: true,
-  change: true,
-  mousedown: true,
-  mouseup: true,
-};
+const allowedEventTypes = [
```

Comment:
An idea for future: maybe we should let user configure which of these they want to enable/disable. So if they are not interested in some events they can save bandwidth.

Reply:
Added to the tracker